src/systems/wind_effects/WindEffects.cc

```diff
@@ -563,6 +570,13 @@ void WindEffectsPrivate::UpdateWindVelocity(const UpdateInfo &_info,
 
   // Update component
   windLinVel->Data() = windVel;
+
+  // Publish wind information (ENU)
+  msgs::Wind windInfo_gz;
+  windInfo_gz.mutable_linear_velocity()->set_x(windVel.X());
+  windInfo_gz.mutable_linear_velocity()->set_y(windVel.Y());
+  windInfo_gz.mutable_linear_velocity()->set_z(windVel.Z());
+  this->wind_pub.Publish(windInfo_gz);
```

Comment:
small optimization to check if there are any subscribers before publishing:

```suggestion
  msgs::Wind windInfo;
  windInfo.mutable_linear_velocity()->set_x(windVel.X());
  windInfo.mutable_linear_velocity()->set_y(windVel.Y());
  windInfo.mutable_linear_velocity()->set_z(windVel.Z());
  if (this->windPub.HasConnections())
    this->windPub.Publish(windInfo);
```